misc/destinations/README.md

```diff
@@ -0,0 +1,235 @@
+# Understanding Destinations in SAP BTP Cockpit
+
+# Overview
+SAP BTP destinations are used to connect to different services and systems in the cloud, onpremise or any publicly available endpoints. They are used to define the connection parameters for the service you want to consume. The destination is a logical representation of the service and contains all the information required to connect to it.
+
+- This guide is focused on consuming OData XML services via SAP BTP destinations, when using [SAP Fiori tools](https://help.sap.com/docs/SAP_FIORI_tools) generator and [Service Centre](https://help.sap.com/docs/bas/sap-business-application-studio/explore-services-using-service-center) in Business Application Studio (BAS).
+- Other destination types are supported, for example, OData SAP HANA XS type services, but this guide is focused on OData XML service only.
+- We are going to use a publicly available endpoint to demonstrate how to configure the SAP BTP destination and how to consume the OData XML service using the SAP Fiori tools generator using different configurations and tools.
+
+# Prerequisites
+
+- You have SAP Cloud Foundry Runtime environment configured in your SAP BTP subaccount
+- You have admin rights to the SAP BTP cockpit to modify destinations
+- Only OData XML services are supported when creating SAP Fiori Elements applications when using SAP Fiori tools generator
+- Understanding [SAP BTP destinations](https://learning.sap.com/learning-journeys/administrating-sap-business-technology-platform/using-destinations)
+- Creating [SAP BTP destinations in the SAP BTP cockpit](https://developers.sap.com/tutorials/cp-cf-create-destination..html)
+
+# Additional Resources
+- Understanding [SAP BTP destination authentication types](https://help.sap.com/docs/connectivity/sap-btp-connectivity-cf/http-destinations)
+
+# Sample Microsoft OData XML service endpoints
+
+The endpoint `https://services.odata.org` exposes a number of OData service endpoints, as shown.
+
+```
+#1
+https://services.odata.org/v2/northwind/northwind.svc/
+#2
+https://services.odata.org/V3/Northwind/Northwind.svc/
+```
+
+# Configuration 
+
+This is a sample SAP BTP destination configuration for the Northwind OData service. The destination name is `northwind` and the URL is `https://services.odata.org`. The authentication type is set to `NoAuthentication`, and the proxy type is set to `Internet`.
+
+![Alt text](northwind_destination.png?raw=true "Screenshot of SAP BTP destination")
+
+The file ![Alt text](northwind?raw=true "SAP BTP destination configuration")can be imported directly into your SAP BTP destinations list and it contains the following properties;
```

Comment:
In the preview this looks like a broken image. Maybe a normal link would be better.
```suggestion
The [file](northwind?raw=true "SAP BTP destination configuration") can be imported directly into your SAP BTP destinations list and it contains the following properties;
```